src/styles/blocks/variable.scss

```diff
@@ -0,0 +1,3 @@
+$sizeicon: 40px;
+$color: #00acdc;
+$starsize: 16px;
```

Comment:
It's a good practice to use more descriptive variable names. Consider changing $starsize to $star-size.